src/wr_led_matrix/src/LEDMatrix.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+#Official ROS Python code tutorial for this thing (has example code, the service server part): https://wiki.ros.org/ROS/Tutorials/WritingServiceClient%28python%29
+#TODO: Import req'd things (rospy, serial, custom service, ...)
+import rospy
+import serial
+from wr_led_matrix.srv import LEDMatrix,LEDMatrixRequest,LEDMatrixResponse
+
+s: serial.Serial
+baud = rospy.get_param("/led_matrix/baud")
+port = rospy.get_param("/led_matrix/com_port")
```

Comment:
These should use a private namespace reference since the name of the node could change based on what's in the launch file.

Reply:
This change has been addressed.